.github/workflows/frontend-e2e-test.yml

```diff
@@ -49,8 +49,9 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 16.x
-          cache: 'yarn'
-          cache-dependency-path: web/yarn.lock
+
+      - name: Install PNPM
+        run: npm i -g pnpm@6.32.11
```

Comment:
Have you checked out this https://github.com/pnpm/action-setup? I think this might be a little better.

But, anyway, let's run through the test first.

Reply:
Let me first say a good news. Yesterday's problem has been solved. The reason is that he didn't use Chrome browser to run it. Just add a setting.

`"For example, I just found that it seems that many errors in file format are related to yarn. Many workflow files have been perfected around this. Then, the pnpm directly downloaded by GitHub does not seem to support node.js12. It describes that the downloaded pnpm does not support NPM below 14 (in fact, you can install a low-level package and ask him to support it, but it feels that it will be troublesome. I am afraid there are more problems.) Only 14 and 16 can be built when building tests. "
![1ec791081ecc16fd7dbe07d687f54ea](https://user-images.githubusercontent.com/72308243/185014045-1194b3bb-3285-4dc4-9229-713d6ccbede4.jpg)
![0ca067b71192e4dc0c2c31de209e9a0](https://user-images.githubusercontent.com/72308243/185014060-21c5f30a-23d7-45ca-a436-3d7be219a1ba.jpg)

"When downloading the relevant environment, it seems that there are some restrictions. Maybe it can be solved by storing it in the warehouse through some link, but I haven't started to look at it. I'm afraid this is also very troublesome. This picture is the reason why the three people are useless. Action is prohibited. It clearly meets the requirements he said, but it can't be used."`

@SkyeYoung I mentioned this problem. You can look at the pictures. If you can solve it, it's best.